### PR TITLE
Update clone sc step to fix pvc bound issue

### PR DESCRIPTION
**Pull request by weherdh**
@liangxia @chao007 Please help to review.
Log: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/39494/console

### CODE REVIEW

File: features/step_definitions/storage_class.rb [hunk 1]
```diff
@@ -189,7 +189,8 @@
      sc_hash.dig("metadata", "annotations", "storageclass.beta.kubernetes.io/is-default-class") == "true"
     ensure_destructive_tagged
   end
-
+  # Update sc to Immediate for smooth testing
+  sc_hash["volumeBindingMode"] = "Immediate"
```
Comment (liangxia) [inline features/step_definitions/storage_class.rb]: We need to put it before `table.raw.each...`, e.g, after `sc_hash["metadata"]["name"] = "#{target_sc}"`

Comment (akostadinov) [reply features/step_definitions/storage_class.rb]: i.e. you want to set a default but be able to override in table. +1

**Comment by weherdh** [comment]
Make sense, updated. Please check again, thanks.

**Comment by liangxia** [comment]
LGTM

**Comment by liangxia** [comment]
LGTM

**Pull request merged**